README.md

```diff
@@ -191,7 +191,32 @@ Turbolinks may not be the only source of page updates in your application. New H
 
 You can handle all of these updates, including updates from Turbolinks page loads, in a single place with the precise lifecycle callbacks provided by [`MutationObserver`](https://developer.mozilla.org/en-US/docs/Web/API/MutationObserver) and [Custom Elements](https://developer.mozilla.org/en-US/docs/Web/Web_Components/Custom_Elements).
 
-In particular, these APIs give you callbacks when elements are attached to and removed from the document. You can use these callbacks to perform transformations and register or tear down behavior as soon as matching elements appear on the page, regardless of how they were added.
+In particular, these APIs give you callbacks when elements are attached to and removed from the document. You can use these callbacks to perform transformations and register or tear down behavior as soon as matching elements appear on the page, regardless of how they were added. For example, if you're checking for elements on the page like so:
+
+``` javascript
+document.addEventListener("turbolinks:load", function() {
+  if (document.getElementById('sparkline')) {
```

Comment:
This is just an example of what is a "bad" way of working with TL. I'm not sure if any further elaboration is required.